docs/source/onboarding_resources/computing/index.rst

```diff
@@ -0,0 +1,359 @@
+#. Include the exact file name and location in the docs. This means if you version up a data file, you will need to update the docs to reflect the new name. This ensures the engineers are aware of any changes. 
+
+For consistency, please use this naming convention for all files: :code:`FILENAME_20230309.ext`. 
+For example, this might be :code:`heart_failure_proportions_20230310.csv` 
+
+.. _cluster_access_bash:
+
+Your Bash Files 
+---------------
+
+Bash files contain commands you would enter into a command line, but 
+specifically ones you will use very often. For example, every time you 
+want to open a Juptyer session, the cluster needs certain information 
+and requirements. To find this information, it looks in your Bash files 
+rather than asking you to enter the same information every time. 
+
+However, Bash files can be confusing since it is less obvious when the 
+information is being used or what it is used for. Therefore, we have 
+provided a copy-and-paste formatting for information to be added to 
+your Bash files. If you would like more information and what this 
+code means and why we need it, this `Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?pageId=83205636#UsingtheCluster&Bash-Bash>`_ has a number of helpful links. 
+
+
+.. todo::
+
+  Confirm and paste in bash files 
+
+.. _conda_environments:
+
+3.0 Conda Environments
+++++++++++++++++++++++
+
+Your environment is a “space” in which you can run code that has some 
+preset information. Environments are helpful as you can preinstall packages 
+into them for easier ability to run programs. GBD has a common environment 
+that you can use, but usually each project will also create its own environment 
+as well. Below are some common questions on environments. 
+
+**What is an environment again?**
+It’s a directory that contains a specific collection of conda packages that 
+you have installed. Basically, it is a shortcut to have all the relevant 
+packages you need for a project in one place. 
+
+**What are the advantages to having separate environments?**
+Over time, Python versioning updates with new information and bug fixes. 
+It can therefore be helpful to create new environments to ensure you have 
+the latest python package versions. 
+
+While you can uninstall and reinstall new versions of packages in existing 
+environments, this can sometimes cause errors in existing code. Therefore, 
+it is helpful to have code exist within environments that have correct 
+versioning and to create new environments with new projects and install the 
+latest versions of packages. 
+
+**How and when should I make a new one?**
+In general, our team has a new environment for each project. When you are 
+starting a new project, you can see if the engineering team has set up an 
+environment for that work. If not, it is best practice to consider setting one up. 
```

Comment:
Wait really? Do others frequently use engineering-created environments? How does this work -- is it read-only to you?

Maybe my last comment doesn't apply then.